src/main/java/at/hannibal2/skyhanni/SkyHanniMod.kt

```diff
@@ -62,6 +63,11 @@ object SkyHanniMod {
         Runtime.getRuntime().addShutdownHook(
             Thread { configManager.saveConfig(ConfigFileType.FEATURES, "shutdown-hook") },
         )
+        if (feature.garden.tracking.tracking) {
```

Comment:
idk if the feature is alr not null here, and this shutdown hook should always get started, not only when the feature is enabled: instead check inside the hook if the feature is enabled